Samples/PushNotifications/cpp-console/cpp-console.cpp

```diff
@@ -133,63 +118,62 @@ int main()
-        // The channel uniquely identifies, this app for this user and device.
-        PushNotificationChannel channel = RequestChannel();
+            // Setup an event handler, so we can receive notifications in the foreground while the app is running.
+            if (channel)
+            {
+                SubscribeForegroundEventHandler(channel);
+            }
+            else
+            {
+                // troubleshooting: Did you replace the zero'ed out remote id (near the top of the sample) with your own?
+                std::cout << std::endl << "There was an error obtaining the Channel Uri" << std::endl;
+            }
 
-        // register the channel, so we can receive notifications in the foreground while the app is running.
-        if (channel)
-        {
-            RegisterForegroundNotificationsHandler(channel);
-        }
-        else
-        {
-            // troubleshooting: Did you replace the zero'ed out remote id (near the top of the sample) with your own?
-            std::cout << "There was an error obtaining the Channel Uri" << std::endl;
+            std::cout << std::endl << "Press 'Enter' at any time to exit App." << std::endl;
+            std::cin.ignore();
         }
+        break;
 
-        std::cout << "Press 'Enter' at any time to exit App." << std::endl;
-        std::cin.ignore();
-    }
-    break;
+        // When it is activated from a push notification, the sample only displays the notification.
+        // It doesn’t register for foreground activation of perform any other actions
+        // because background activation is meant to let app perform only small tasks in order to preserve battery life.
+        case ExtendedActivationKind::Push:
+        {
+            PushNotificationReceivedEventArgs pushArgs = args.Data().as<PushNotificationReceivedEventArgs>();
 
-    // When it is activated from a push notification, the sample only displays the notification.
-    // It doesn�t register for foreground activation of perform any other actions
-    // because background activation is meant to let app perform only small tasks in order to preserve battery life.
-    case ExtendedActivationKind::Push:
-    {
-        PushNotificationReceivedEventArgs pushArgs = args.Data().as<PushNotificationReceivedEventArgs>();
+            // Call GetDeferral to ensure that code runs in low power
+            auto deferral = pushArgs.GetDeferral();
 
-        // Call GetDeferral to ensure that code runs in low power
-        auto deferral = pushArgs.GetDeferral();
+            auto payload = pushArgs.Payload();
 
-        auto payload = pushArgs.Payload();
+            // Do stuff to process the raw notification payload
+            std::string payloadString(payload.begin(), payload.end());
+            std::cout << std::endl << "Push notification content received from BACKGROUND: " << payloadString.c_str() << std::endl;
```

Comment:
Similar as in the unpkg sample PR. Any reason for `std::cout << std::endl << "Text"` ?

Reply:
Similar answer. I've replaced them with \n.